### ISSUE TITLE
Command line examples not working in v0.6.3 

### ISSUE THREAD

**Issue by fforster** [thespacedoctor/panstamps#9]
None of the command line examples shown in the documentation work after release 0.6.3. It seems to be a problem with the new end point to collect image stamps.

```
(base) fforster@fforster-XPS-13-9310:~$ panstamps stack 09:55:52.2 +69:40:47
* 10:44:55 - WARNING: /home/fforster/anaconda3/lib/python3.8/site-packages/panstamps/downloader.py:get:236 > No images found. Your options may not be set correctly. Here's the requested URL:
http://ps1images.stsci.edu/cgi-bin/ps1cutouts?pos=09%3A55%3A52.2+%2B69%3A40%3A47&filter=g&filter=r&filter=i&filetypes=stack&size=240&output_size=1200&verbose=0&autoscale=99.500000&catlist=

(base) fforster@fforster-XPS-13-9310:~$ panstamps --width=4 --filters=griyz stack 09:55:52.2 +69:40:47
* 10:46:15 - WARNING: /home/fforster/anaconda3/lib/python3.8/site-packages/panstamps/downloader.py:get:236 > No images found. Your options may not be set correctly. Here's the requested URL:
http://ps1images.stsci.edu/cgi-bin/ps1cutouts?pos=09%3A55%3A52.2+%2B69%3A40%3A47&filter=g&filter=r&filter=i&filter=y&filter=z&filetypes=stack&size=960&output_size=1200&verbose=0&autoscale=99.500000&catlist=

(base) fforster@fforster-XPS-13-9310:~$ panstamps -Fj --width=4 --filters=gri --downloadFolder=./ stack 09:55:52.2 +69:40:47
* 10:46:34 - WARNING: /home/fforster/anaconda3/lib/python3.8/site-packages/panstamps/downloader.py:get:236 > No images found. Your options may not be set correctly. Here's the requested URL:
http://ps1images.stsci.edu/cgi-bin/ps1cutouts?pos=09%3A55%3A52.2+%2B69%3A40%3A47&filter=g&filter=r&filter=i&filetypes=stack&size=960&output_size=1200&verbose=0&autoscale=99.500000&catlist=

(base) fforster@fforster-XPS-13-9310:~$ panstamps -FjAt --width=4 --filters=gri --downloadFolder=./ stack 09:55:52.2 +69:40:47
* 10:46:48 - WARNING: /home/fforster/anaconda3/lib/python3.8/site-packages/panstamps/downloader.py:get:236 > No images found. Your options may not be set correctly. Here's the requested URL:
http://ps1images.stsci.edu/cgi-bin/ps1cutouts?pos=09%3A55%3A52.2+%2B69%3A40%3A47&filter=g&filter=r&filter=i&filetypes=stack&size=960&output_size=1200&verbose=0&autoscale=99.500000&catlist=
```

**Comment by thespacedoctor** [comment]
Hi Francisco - want to test out the new release. Should be working again now.

**Comment by fforster** [comment]
Hi, I am trying to run panstamps from google colab for a tutorial, but I cannot download data using the command line option. Any idea why this could be? Thanks!


!panstamps --width=2 --filter=r stack 20.404016916666667 -22.60269873333333
output
Please add any require settings to file '/root/.config/panstamps/panstamps.yaml' before proceeding to run panstamps.
Traceback (most recent call last):
  File "/usr/local/bin/panstamps", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/panstamps/cl_utils.py", line 62, in main
    su = tools(
  File "/usr/local/lib/python3.10/dist-packages/fundamentals/tools.py", line 250, in __init__
    stream = open(self.arguments["settingsFile"], 'r')
FileNotFoundError: [Errno 2] No such file or directory: '/root/.config/panstamps/panstamps.yaml'

**Comment by thespacedoctor** [comment]
Hi Francisco, can you try and run `panstamps init` and then run your commands again? If that doesn't work I may have broken something inadvertently in my `fundamentals` code.

**Comment by fforster** [comment]
Hi, panstamps init works, but I cannot download new data. Perhaps the command line options changed?
![image](https://github.com/thespacedoctor/panstamps/assets/5090828/0d216a2e-b348-4ea0-9842-03c21370a674)

**Comment by thespacedoctor** [comment]
I've pushed what I hope is a fix to PyPi (v0.6.6). Want to upgrade and try again?
